### PR TITLE
feat: (migration)Add controller to approve a comment as an answer

### LINKED ISSUE #59: Migrate comment approvedBy controller.
Create a controller with the [approvedBy](https://github.com/wizeline/wize-q-back/blob/b197fde689bdbf82a8e70376ccfe833b7e863be2/controllers/comments.js#L154) action.

### PR DESCRIPTION
**Pull request by GeorginaFraire**
#### What does this PR do?
- Add controller to approve a comment as answer.

#### Where should the reviewer start?
 - app/controllers/comments/approvedBy.js
 - tests/integration/controllers/comments/approvedComment.test.js

#### How should this be manually tested?
Run the test integration. `npm run test:integration`

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #59 

#### Screenshots
Screen Shot 2022-11-01 at 17 50 46

### CODE REVIEW

File: app/utils/backend/objectUtils.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function isEmpty (object) {
```
Comment (Hoosep) [inline app/utils/backend/objectUtils.js:1]: I suggest to rename this function like isEmptyObject, just to emphasize it.

**Review by Hoosep — APPROVED** [review]

**Review by Hoosep — APPROVED** [review]
LGTM! :D

File: app/controllers/comments/approvedBy.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { db } from "~/utils/db.server";
+import { approvedByCommentSchema } from "~/utils/backend/validators/comments";
+import { INVALID_PARAMS_FOR_OPERATION_ERROR_MESSAGE } from "~/utils/constants";
+import { COMMENT_AS_AN_ANSWER, DEFAULT_ERROR_MESSAGE, UPDATE_COMMENT_ERROR_MESSAGE} from '~/utils/backend/constants';
```
Comment (albertcastaned) [inline app/controllers/comments/approvedBy.js:4]: Remove/use unused variable: 

`'UPDATE_COMMENT_ERROR_MESSAGE'` is defined but never used`

**Review by albertcastaned — APPROVED** [review]

**Review by AldoDominguezCha — COMMENTED** [review]
Looks really good! 🎉

**Comment by AldoDominguezCha** [comment]
Just the same as Alberto already mentioned, can we remove the unused UPDATE_COMMENT_ERROR_MESSAGE constant in the controller? Otherwise it looks really good 🎉

**Review by AldoDominguezWizeline — APPROVED** [review]
LGTM! 🎉 Just please remove the unused constant in the controller, otherwise looks great!

File: app/controllers/comments/approvedBy.js (hunk 2 above)
Comment (GeorginaFraire) [reply app/controllers/comments/approvedBy.js]: Done

**Review by albertcastaned — APPROVED** [review]

**Pull request merged**